src/quo2/components/community_card_view.cljs

```diff
@@ -0,0 +1,231 @@
+                                          background-color] :as community}]
+  [react/view {:flex    1}
+   [react/view {:flex-direction     :row
+                :border-radius      border-radius
+                :padding-horizontal 12
+                :align-items        :center
+                :padding-vertical   8
+                :background-color   background-color}
+    [react/view
+     [communities.icon/community-icon-redesign community 32]]
+    [react/view {:flex              1
+                 :margin-horizontal 12}
+     [text/text {:weight              :semi-bold
+                 :size                :paragraph-1
+                 :accessibility-label :community-name-text
+                 :number-of-lines     1
+                 :ellipsize-mode      :tail}
+      name]
+     [community-stats-column :list-view]]
+    (when (= status "gated")
+      [permission-tag-container {:locked       locked
+                                 :status       status
+                                 :tokens       tokens}])]])
+
+(defn communities-membership-list-item [{:keys [id name status tokens locked] :as community}]
+  [react/view {:margin-bottom       12
+               :padding-horizontal  8}
+   [react/touchable-highlight {:underlay-color      colors/primary-50-opa-5
+                               :style               {:border-radius 12}
+                               :on-press      (fn []
+                                                (>evt [::communities/load-category-states id])
+                                                (>evt [:dismiss-keyboard])
+                                                (>evt [:navigate-to :community {:community-id id}]))
+                               :on-long-press #(>evt [:bottom-sheet/show-sheet
+                                                      {:content (fn []
+                                                                  [community/community-actions community])}])}
+    [react/view {:flex               1
+                 :padding-vertical   8
+                 :padding-horizontal 12}
+     [react/view {:flex-direction    :row
+                  :border-radius     16
+                  :align-items       :center}
+      [communities.icon/community-icon-redesign community 48]
+      [react/view {:margin-left   12
+                   :flex          1}
+       [community-title {:title  name}]]
+      (when (= status "gated")
+        [react/view {:justify-content   :center
+                     :margin-right      12}
+         [permission-tag-container {:locked       locked
+                                    :status       status
+                                    :tokens       tokens}]])]]]])
+
+(defn community-card-view [{:keys [id name description]  :as community}]
+  (let [{:keys [tags locked section status cover tokens]}
+        (get communities-items-featured :data)
+        window-width (* (<sub [:dimensions/window-width]) 0.90)]
```

Comment:
could you elaborate? im not sure we should use `window-width`

Reply:
the community-card-view when used in a horizontal scroll view needs to have a width property otherwise it extend beyond the view

Reply:
I avoided using pixels as it looks different than the design on different devices

Reply:
You can try using `:flex` with padding or margin?

Reply:
looks like the `:flex` doesnt work for  list-item width in a horizontal flat-list

Reply:
in this case a list-item used in a horizontal flat-list looks like should have the width property as `:flex` doesnt seem to work. This is as explained here https://snack.expo.dev/H1-wnC5HM